cv/qc-cv.obo

```diff
@@ -18,1967 +18,341 @@ import: https://raw.githubusercontent.com/ISA-tools/stato/dev/releases/1.4/stato
+relationship: has_units STATO:0000300 ! dimensionless ratio
 
 [Term]
 id: MS:4000077
-name: Area under TIC
-def: "The area under the total ion chromatogram." [PSI:QC]
+name: AA sequence length
+def: "The length of an amino acid sequence." [PSI:QC]
 is_a: MS:4000003 ! single value
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:1000235 ! total ion current chromatogram
-relationship: has_relation MS:4000013 ! single run based
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000078
-name: Area under TIC RT quantiles
-def: "The area under the total ion chromatogram of the retention time quantiles. Number of quantiles are given by the n-tuple." [PSI:QC]
-is_a: MS:4000004 ! n-tuple
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:1000235 ! total ion current chromatogram
-relationship: has_relation MS:4000013 ! single run based
+name: protein coverage table
+def: "The coverage of protein sequences from the peptide sequences identified. The table records the coverage itself (in percent), the protein accession (SHOULD correspond to the accession used in fasta document used for Sequence DB based identification), length of the protein (, and optionally if it is 'target' or 'decoy' entry and how many PSM can be attributed to this protein)." [PSI:QC]
+comment: The protein coverage can provide insight into the sensitivity of the instrument/identification/protocols used.
+is_a: MS:4000005 ! table
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_column MS:1001093 ! sequence coverage
+relationship: has_column MS:1000885 ! protein accession
+relationship: has_column PEFF:0001006 or term above ???
+relationship: has_optional_column MS:1001197 ! DB composition target+decoy ???
+relationship: has_optional_column MS:1002153 ! protein level PSM counts
 
 [Term]
 id: MS:4000079
-name: Pump pressure chromatogram
-def: "Representation of chromatographic pressure versus time. The first column contains the retention time points in seconds, the second column the corresponding pressures in pascal." [PSI:QC]
-is_a: MS:4000006 ! table
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:1003019 ! pressure chromatogram
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000261 ! Pressure
+name: sampling rates
+def: "The sampling rates of identified MS2 and their respective frequency. The table records the sampling rate (i.e. once=1, twice=2, ...) in one column and the respective frequency of that sampling rate in another column. Optionally, another column giving the sequences of peptides sampled (space separated) can be present." [PSI:QC]
```

Comment:
the name should be more specific
and I'm not sure what the metric means and how its computed :)
My guess: one row per peptide sequence, where the sampling rate says how many times an eluting peptide was triggered for MS2 (i.e. potential oversampling).
But would you include unidentified MS2 spectra from the same mass trace? (probably not, also a bit harder to compute)

how do you compute the frequency? What is the denominator?

While this metric is nice, it will be a BIG table.
How about a separate metric which just holds some aggregration stats?
See https://htmlpreview.github.io/?https://github.com/cbielow/PTXQC/blob/master/inst/examples/report_v1.0.10__txt_5files_withMatch-100min.html#evd-oversampling-msms-counts-per-3d-peak